src/framework/components/light/component.js

```diff
@@ -178,6 +54,841 @@ class LightComponent extends Component {
         this._cookieMatrix = null;
     }
 
+    /**
+     * @type {import('./data.js').LightComponentData}
+     */
+    get data() {
+        const record = this.system.store[this.entity.getGuid()];
+        return record ? record.data : null;
+    }
```

Comment:
Isn't this defined in `Component` superclass? Why do we need this anyway? The new component architecture doesn't use the data block. All component data is now stored on the component itself.

Reply:
This was purely to define the type of data and there is no way to to override a `getter`